lib/govuk_index/presenters/indexable_content_presenter.rb

```diff
@@ -34,7 +35,7 @@ def hidden_content
     def indexable_content_parts
       indexable_content_keys.flat_map do |field|
         indexable_values = details[field] || []
-        field == 'parts' ? parts(indexable_values) : [indexable_values]
+        field == 'parts' ? parts(indexable_values) : Array.new(indexable_values)
```

Comment:
Was this broken before? It looks like `indexable_values` has always been an array.